.github/scripts/make-docs.sh

```diff
@@ -3,4 +3,4 @@ source $(dirname ${BASH_SOURCE[0]})/common.sh
 ci_conda_activate
 cd docs
 source install-deps
-make linkcheck
+make docs
```

Comment:
Since link checking is now done on the generated HTML docs, and is part of the `make docs` target, run `make docs` instead of the obsolete `make linkcheck`.